client/auth/login.jsx

```diff
@@ -96,8 +117,13 @@ module.exports = React.createClass( {
 		return this.hasLoginDetails();
 	},
 
+	toggleSelfHostedInstructions: function () {
+	    var isShowing = !this.state.showingInstructions;
+	    this.setState({ showingInstructions: isShowing });
```

Comment:
Minor: space before/after the object: `this.setState( { showingInstructions: isShowing } );`